x-pack/plugins/maps/public/actions/visualize_geo_field_action.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import rison from 'rison-node';
+import uuid from 'uuid/v4';
+import { stringify } from 'query-string';
+import { i18n } from '@kbn/i18n';
+import { createAction } from '../../../../../src/plugins/ui_actions/public';
+import {
+  createKbnUrlStateStorage,
+  IKbnUrlStateStorage,
+} from '../../../../../src/plugins/kibana_utils/public';
+import { DiscoverAppState } from '../../../../../src/plugins/discover/public';
+import { getApplication, getIndexPatternService } from '../kibana_services';
+
+export const ACTION_VISUALIZE_GEO_FIELD = 'ACTION_VISUALIZE_GEO_FIELD';
+
+export const visualizeGeoFieldAction = createAction<typeof ACTION_VISUALIZE_GEO_FIELD>({
+  type: ACTION_VISUALIZE_GEO_FIELD,
+  getDisplayName: () =>
+    i18n.translate('xpack.maps.discover.visualizeFieldLabel', {
+      defaultMessage: 'Visualize on Kibana Maps',
```

Comment:
:art: I think since we're already in Kibana, I'd change that title just to "Visualize in Maps" or "Visualize on a map", and get rid of the "Kibana" here, for a briefer text. @thomasneirynck @nreese What are your thoughts?

Reply:
Agree with Tim, "Visualize in Maps" is better.

Reply:
I have changed it to `Visualize on a map`. If there are objections on this let me know